skfuzzy/fuzzymath/fuzzy_ops.py

```diff
@@ -661,6 +661,60 @@ def interp_membership(x, xmf, xx):
     return xxmf
 
 
+def interp_value(x, xmf, y):
+    """
+    Finds the degree of membership ``u(xx)`` for a given value of ``x = xx``.
+
+    Parameters
+    ----------
+    x : 1d array
+        Independent discrete variable vector.
+    xmf : 1d array
+        Fuzzy membership function for ``x``.  Same length as ``x``.
+    y : float
+        Membership function value at ``x``.
+
+    Returns
+    -------
+    xx : 1d array
+        Array of discrete singleton values on universe ``x`` whose
+        membership function value is y, ``u(xx[i])==y``.
+
+    Notes
+    -----
+    For use in Fuzzy Logic, where an interpolated discrete value ``xx`` which its
```

Comment:
This function complements ``interp_membership``. Whereas ``interp_membership`` accepts an arbitrary value on the universe and returns its interpolated fuzzy membership, ``interp_value`` does the opposite. This function accepts an arbitrary fuzzy membership value and returns the universe value(s) which would produce it.